build.sh

```diff
@@ -14,6 +14,35 @@
 # limitations under the License.
 # ****************************************************************************
 
+set -ex
+
+# Install Mono:
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get -y install gnupg ca-certificates
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+echo "deb https://download.mono-project.com/repo/ubuntu stable-bionic/snapshots/6.0.0.319 main" | tee /etc/apt/sources.list.d/mono-official-stable.list
+apt update
+DEBIAN_FRONTEND=noninteractive apt -y install mono-devel=6.0.0.*
+
+# Install .NET Core SDK:
+DEBIAN_FRONTEND=noninteractive apt-get -y install software-properties-common wget
+wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+dpkg -i packages-microsoft-prod.deb
+add-apt-repository universe
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get -y install apt-transport-https
+DEBIAN_FRONTEND=noninteractive apt-get -y install dotnet-sdk-3.1
+
+# Install .NET Core 1.1 runtime:
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get -y install libunwind-dev
+cd /usr/share/dotnet
+wget https://download.visualstudio.microsoft.com/download/pr/d3bbce9c-a15e-4224-a271-064546459e53/694a1c880d7140c80215acb5dceab3db/dotnet-ubuntu.18.04-x64.1.1.13.tar.gz -O dotnet-ubuntu.18.04-x64.1.1.13.tar.gz
+tar xvzf dotnet-ubuntu.18.04-x64.1.1.13.tar.gz
```

Comment:
It might likewise be possible to build and run using `dotnet` Docker images. While not necessary right now, it might be a relatively easy way to decouple ourselves from whatever Appveyor decides to preinstall, while keeping it relatively simple to choose a different SDK / runtime version.

Reply:
This turns out to be a little more complicated because we'd run a different container for building with the .NET Core SDK 3.1 (`mcr.microsoft.com/dotnet/core/sdk:3.1`) than when running with the .NET Core 1.1 runtime (`mcr.microsoft.com/dotnet/core/runtime:1.1`), and things can go wrong if packages restored by the former aren't seen by the latter; we'd have to at least map a NuGet package restore directory.